src/libs/actions/Report.js

```diff
@@ -147,7 +147,9 @@ function getChatReportName(sharedReportList) {
  */
 function getSimplifiedReportObject(report) {
     const reportActionList = lodashGet(report, ['reportActionList'], []);
-    const lastReportAction = !_.isEmpty(reportActionList) ? _.last(reportActionList) : null;
+    const lastReportAction = !_.isEmpty(reportActionList)
+        ? _.find([...reportActionList].reverse(), reportAction => reportAction.message.html !== '')
```

Comment:
You use `reportAction` here but `reportActionItem` further down. Either is fine but it would be good to be consistent

Reply:
The reason the `reportActionItem` is used further down is because there is already a `reportAction` in upper scope. But I believe I can do even better with destructuring. I'll commit the suggested change, please take a look.